cwl/src/main/scala/cwl/CommandOutputBinding.scala

```diff
@@ -63,9 +67,13 @@ case class CommandOutputBinding(
 
   private def load64KiB(path: String, ioFunctionSet: IoFunctionSet): String = {
     // This suggests the IoFunctionSet should have a length-limited read API as both CWL and WDL support this concept.
-    // val content = ioFunctionSet.readFile(path)
-    val content = better.files.File(path).bytes.take(64 * 1024).toArray
-    new String(content)
+    // ChrisL: But remember that they are different (WDL => failure, CWL => truncate)
+    val content = ioFunctionSet.readFile(path)
+
+    // TODO: propagate IO, Try, or Future or something all the way out via "commandOutputBindingtoWdlValue" signature
+    // TODO: Stream only the first 64 KiB, this "read everything then ignore most of it" method is terrible
```

Comment:
yes, I'd even say this could be a ticket on top of a TODO

Reply:
https://github.com/broadinstitute/cromwell/issues/2771